src/doc/antora.yml

```diff
@@ -1,4 +1,4 @@
 name: pensjon-selvbetjening-opptjening-frontend
 title: Selvbetjening-Opptjening-Frontend
 version: master
-start_page: doc.adoc
+start_page: ROOT:doc.adoc
```

Comment:
Er dette feil? ref https://github.com/navikt/pensjon-selvbetjening-opptjening-frontend/pull/87/files

Reply:
Det var feil først, men referering til filen fungerte likevel ikke, selv om den er i samme directory som filen som referer til den. Derfor prøver vi med å faktisk opprette ROOT-mappa som manglet sist og ser om det hjelper.

Reply:
Nice! Ligger det beskrevet i confluence?

Reply:
Har ikke skrevet noe i confluence. Etter oppretting av en mappestruktur som passet antora bedre fikk jeg dette til å virke. Men uheldigvis så krever antora at alle adoc-filene ligger på samme sted, noe som bryter med prinsippet vi har at adoc-filen for hver modul skal ligge sammen med koden for moduelen. Så jeg vet ikke helt hvordan jeg skal løse dette.
Siden ble seende slik ut: https://friendly-chainsaw-5bf7db7a.pages.github.io/pensjon-selvbetjening-opptjening-frontend/doc.html (man kan se at den ikke finner filene som ikke ligger i modules/root)

Reply:
I samme mappe eller er det lov med anotra mappe med undermapper?

Reply:
Er vel bare å lage en mappe som heter pages over adoc filene?
https://docs.antora.org/antora/latest/pages-directory/

Reply:
Ja, det er det som må til for å få det til å virke. Jeg vet hvordan eg får dette til å fungere. Ulempen er at vi har adoc-filene slik at men i pakken for en komponent finner selve koden til komponenten og dokumentasjonen. Dermed vet man med en gang hvilken adoc-fil man skal oppdatere når man har gjort endringer på komponenten. Når man samler alle adoc-filenne mister man denne muligheten.

Reply:
@r154508 skal https://friendly-chainsaw-5bf7db7a.pages.github.io/pensjon-selvbetjening-opptjening-frontend/doc.html fungere nå?